scripts/quickstart.sh

```diff
@@ -86,7 +98,7 @@ do
     --http-address              0.0.0.0:1919${i} \
     --prometheus.url            http://localhost:909${i} \
     --tsdb.path                 data/prom${i} \
-    --gcs.bucket                "${GCS_BUCKET}" \
```

Comment:
Have you tested it maybe? (:

Reply:
well. i did run the script...  but without prometheus and thanos installed, so those commands directly did fail because of that! :)
Using bash -x, the output looks good.

I'm really using containers and used this script when starting as way to better understand the different parts and how they connect to each other